website/docs/reference/exposure-properties.md

```diff
@@ -54,6 +51,42 @@ exposures:
 ```
 </File>
 
+</VersionBlock>
+
+<VersionBlock lastVersion="1.3">
+
+<File name='models/<filename>.yml'>
+
+```yml
+version: 2
+
+exposures:
+  - name: <string_with_underscores>
+    [description](description): <markdown_string>
+    type: {dashboard, notebook, analysis, ml, application}
+    url: <string>
+    maturity: {high, medium, low}
+    [tags](resource-configs/tags): [<string>]
+    [meta](resource-configs/meta): {<dictionary>}
+    owner:
+      name: <string>
+      email: <string>
+    
+    depends_on:
+      - ref('model')
+      - ref('seed')
+      - source('name', 'table')
+      
+    # added in dbt Core v1.3
+    label: "Human-Friendly Name for this Exposure!"
+    [config](resource-properties/config):
+      enabled: true | false
+
+  - name: ... # declare properties of additional exposures
+```
+</File>
+
+</VersionBlock>
 
 ## Example
```

Comment:
Let's update the first example here to exactly match the one in `docs/build/exposures.md`. (Could even use a snippet!)